tests/rig/Cargo.toml

```diff
@@ -18,7 +18,7 @@ forward_system = { path = "../../forward_system" }
 oracle_provider = { path = "../../oracle_provider" }
 zk_ee = { path = "../../zk_ee" }
 zksync_os_runner = { path = "../../zksync_os_runner"}
-system_hooks = { path = "../../system_hooks" }
+system_hooks = { path = "../../system_hooks", features = ["p256_precompile"]}
```

Comment:
Let's not include it here, but just for instances/precompiles. We might need to define a feature in rig to forward it. Otherwise, EVM state tests will fail